terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/logging/LoggingTest.java

```diff
@@ -50,7 +50,7 @@ public void test01_01_createDefaultXTrackMDC() {
 
         // cut x-Track MDC
         String targetMdc = driver.findElement(By.id("xTrackMDC")).getText();
-        String footerMdc = driver.findElement(By.cssSelector("p")).getText();
+        String footerMdc = driver.findElement(By.id("xtrack")).getText();

```

Comment:
@kazuki43zoo @btyoshikawaa 
Thank you for your comments.
Taking all into account, I can think three solutions, shown below.
1. add `WebDriverOperations` class to blank project(and add [`getText()`](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/master/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/welcome/HelloIT.java#L40) method and [`displayPage()`](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/master/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/welcome/HelloIT.java#L38) method) and `terasoluna-gfw-functionaltest` application’s `WebDriverOperations` class fit to `spring-functionaltest` application’s.
2. only `terasoluna-gfw-functionaltest` application’s `WebDriverOperations` class fit to `spring-functionaltest` application’s.
3. not to do anything by observing the policy to directly use the WebDriver's API except for complex logic.

Please tell me which solution do you think best?
I think... if 1st or 2nd solution choose, I think in `terasoluna-gfw-functionaltest` application's anather method should move `WebDriverOperations` class.
 ex. [`click()`](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/date/DateTest.java#L44) method , [`getCurrentUrl()`](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java#L200) method , [findElement()](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/codelist/CodeListTest.java#L39) method … and more?